Sources/HaCWebsiteLib/Controllers/EventApiController.swift

```diff
@@ -3,26 +3,39 @@ import KituraNet
 import SwiftyJSON
 import Foundation
 import DotEnv
+import LoggerAPI
 
 struct EventApiController {
   static var handler: RouterHandler = { request, response, next in
     response.headers["Content-Type"] = "text/plain; charset=utf-8"
     guard let parsedBody = request.body else {
       next()
       response.statusCode = HTTPStatusCode.internalServerError
+      // TODO: Log info about why?
```

Comment:
The plan is to complete these TODOs before merging, so please make sure that they are all removed (or converted to issues) before any approval!